src/components/ImportExport.js

```diff
@@ -151,22 +152,36 @@ class ImportExport extends React.Component {
     const { selectedExport, exportType } = this.state;
 
 
-    const trailFeatures = _.flatMap(_.values(trails.toJS()), item => item.features);
-    const hydrantFeatures = _.flatMap(_.values(hydrants.toJS()), item => item.feature);
-    const ext = exportType === 'GJ' ? 'json' : 'kml';
-
-    hydrantFeatures.forEach((f,i) => {
-      const desc = `${f.get('trailName')},${i},${f.get('name')}`
-      f.set('description', desc)
+    const trailFeatures = []
+    const hydrantFeatures = []
+
+    trails.valueSeq().forEach((v) => {
+      let trailName = v.get('name').split(' ').join('_')
+      // Iterate through Trail's Features
+      v.get('features').forEach((f) => {
+        if (f.get('originalTrailName')) {
+          trailName = f.get('originalTrailName')
+        }
+        const description = trailName
+        f.unset('features')
```

Comment:
This isn't great. We shouldn't have things we don't want like this added to the trail when it's modified and we have to take off again here. Can you modify the TrailReducer so it's like HydrantReducer's `updateHydrantFeatures` and only puts the attributes we want on the feature?

Reply:
Updated in clean up branch.